table/views.py

```diff
@@ -10,6 +10,7 @@
 
 from table.forms import QueryDataForm
 from table.tables import TableDataMap
+from functools import reduce
```

Comment:
Seems like it isn't to me: https://github.com/shymonk/django-datatable/blob/cd5160929a795c6d7528fcc5f15082b8d61b55c9/table/views.py#L72